s32351-pj-BartoszWaszczyk/pamietnik.txt

```diff
@@ -0,0 +1,20 @@
+2023-09-30
+Hmmm. Cudnie, bardzo lubię ćwiczenia w terminalu. Zawsze kojaerzyło mi się i nadal kojarzy z posiadaniem wiedzy tajemnej, któa pozwala na robienie cudów. Oczywiście prócz romnantyzmu hakerstwa, praktyczna strona dzisuejszego dnia pozwoliła mi ugruntowac wiedzę ze sposobów zmiany katalogu, odczytywania informacji o strukturze i zawartości systemu plików. Gdzie i jakie pliki się znajdują, za co odpowiada zawartość folderu (oczywicie mówimy o linuchu jak na razie), czy na przykłąd jak tworzyć pliki i zapełniać je treścią. Zabawy na cli (bashu) etc. A także polecenie man (dotychczas szczytem dla mnie było użycie przełącznika -h / --help).
+
+2023-10-14
+dziś - uprawnienia. Temat mi znany baaaardzo 'po łebkach' jak dotąd; od teraz wiem już jak i co zmieniać, żeby całkowicie się zakopać w katalogu, do którego uprawnień żadnych nie mam, tak jak i do nadrzędnych i nadrzędnych tych nadrzędnych... Ale poważnie, odkryciem dla mnie jest, że zmieniac moge uprawnienia do katalogu bezpośrednio leżącego pod tym, do którego mamy uprawnienia zapisu. A dla leżącego jeszcze głębiej nie mogę dokonywac zmian, dopóki nie nadam/przywrócę sobie uprawnienia do odczytu, zapisu i wejscia. Jeszcze tego w pełni nie testowałem --> todo. Wreszcie rozumiem o co biega w zapisie rwx/7/111 (opisowo/oktalnie/w kodzie dwójkowym). Maska --> Defaulty to coś co pozwala na ograniczenie powtarzania wpisywania tego samego wielokrotnie, więc dla mnie bomba. Ale irytuje mnie podejście: 'dawaj minimum koniecznych uprawnień, nigdy więcej niż potrzeba'. Akurat na co dzień pracuję 'po drugiej stronie barykady'i rzygam już 'Service NOW' i kolejnym ticketem w którym po trzykroć muszę udowadniać po co mi update do ulubionego (i zatwierdzonego!) edytora tekstów (Sublime Text akurat w moim przypdku).
```

Comment:
Na niestrawności proponuję wizytę u gastrologa :D
A na poważnie:
ServiceNOW... been there... seen it... went over it.
Osobiście także lubię Sublime Text.

Irytacja wynika z braku wygody a wygoda prawie zawsze jest przeciwieństwem bezpieczeństwa.
Przykład:
- jak uzyskać podwyższone uprawniena less-em
- jak uzyskać podwyższone uprawniena vim-em
- jak działa setuid i stickybit-em?
- dlaczego partycję hibernacji i wymiany należy szyfrować jeśli już zabieramy się za szyfrowanie danych?
- dlaczego nie należy rzeczy uruchomionych w kontenerach uruchamiać jako root na kontenerze?

Na te i wiele innych pytań administratorzy znają dobre przykłady i wyjaśnienia.
Ostatecznie nie wzięła się z nikąd ta sugestia by nadawać minimalne uprawnienia/prawa.